src/content/protocols/uniswap-v3.md

```diff
@@ -0,0 +1,117 @@
+| ProxyAdmin                         | 0xB753548F6E010e7e680BA186F9Ca1BdAB2E90cf2 |
+| TickLens                           | 0xbfd8137f7d1516D3ea5cA83523914859ec47F573 |
+| Quoter                             | 0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6 |
+| SwapRouter                         | 0xE592427A0AEce92De3Edee1F18E0157C05861564 |
+| NFTDescriptor                      | 0x42B24A95702b9986e82d421cC3568932790A48Ec |
+| NonfungibleTokenPositionDescriptor | 0x91ae842A5Ffd8d12023116943e72A606179294f3 |
+| TransparentUpgradeableProxy        | 0xEe6A57eC80ea46401049E92587E52f5Ec1c24785 |
+| NonfungiblePositionManager         | 0xC36442b4a4522E871399CD717aBDD847Ab11FE88 |
+| V3Migrator                         | 0xA5644E29708357803b5A882D272c41cC0dF92B34 |
+
+## Permission owners
+
+| Name                        | Account                                                                            | Type                |
+| --------------------------- | ---------------------------------------------------------------------------------- | ------------------- |
+| ProxyAdmin                  | [address](https://etherscan.io/address/0x1a9C8182C09F50C8318d769245beA52c32BE35BC) | TimeLock Contract   |
+| TransparentUpgradeableProxy | [address](https://etherscan.io/address/0xB753548F6E010e7e680BA186F9Ca1BdAB2E90cf2) | ProxyAdmin Contract |
+| GovernerBravo               | [address](https://etherscan.io/address/0x408ED6354d4973f66138C91495F2f2FCbd8724C3) | Contract            |
+
+## Permissions
+
+| Contract                         | Function          | Impact                                                  | Owner                                   |
+| -------------------------------- | ----------------- | ------------------------------------------------------- | --------------------------------------- |
+| UniswapV3Factory                 | setOwner          | changes the owner to a new address                      | TimeLocked DAO contract (GovernorBravo) |
+| UniswapV3Factory                 | enableFeeAmount   | enables a fee amount for a given tick spacing           | TimeLocked DAO contract (GovernorBravo) |
+| UniswapV3Factory (UniswapV3Pool) | setFeeProtocol    | set the protocol's % share of the fees                  | TimeLocked DAO contract (GovernorBravo) |
+| UniswapV3Factory (UniswapV3Pool) | collectProtocol   | withdraw accumulated protocol fees to custom address    | TimeLocked DAO contract (GovernorBravo) |
+| ProxyAdmin                       | renounceOwnership | abandons ownership to the contract                      | TimeLocked DAO contract (GovernorBravo) |
+| ProxyAdmin                       | changeProxyAdmin  | changes ownership to the contract                       | TimeLocked DAO contract (GovernorBravo) |
+| ProxyAdmin                       | upgrade           | upgrades the contract                                   | TimeLocked DAO contract (GovernorBravo) |
```

Comment:
can you write a potential impact on upgrading, you rightfully wrote in the overview upgradability section that it just impacts the tokenURI, but it should be here as well

+ make full sentences and if it's admin management (transfer, revoke etc.) reference the core permission it affects `upgrade` e.g